lib/systems/examples.nix

```diff
@@ -112,11 +112,26 @@ rec {
     config = "aarch64-none-elf";
     libc = "newlib";
   };
+  
+  aarch64be-embedded = {
```

Comment:
@Ericson2314 do we have any requirements for systems added in this file?

Reply:
Not really! The only thing would be trying to keep the `config`s unique.